.mergify.yml

```diff
@@ -16,7 +16,6 @@ pull_request_rules:
         add: ["type: breaking"]
   - name: merge Scala Steward's PRs
     conditions:
-      - base=master
```

Comment:
Should it be changed to `base=develop` ?

Reply:
Does it even need to be there? :shrug: It's one less configuration that, as far as I can see, isn't needed and that could potentially be wrong, like in our case here.

Reply:
True, steward triggers it only on default branch.